Service Tasks/best-practices-and-preparation-for-a-virtual-machineovfova-import.md

```diff
@@ -1,28 +1,43 @@
-* Validate the [Guest Operating System is supported on the CenturyLink Cloud for Import](../Service Tasks/supported-guest-operating-systems-for-ovaovf-import.md)
-* If exporting multiple Virtual Machines or a collection of Virtual Machines (such as a vAPP), it is recommended to export each VM individually. Individual files provide greater flexibility and reduce repeated upload in the event of file corruption (re-uploading a single server vs. the entire collection).</p>
-* To reduce file transfer time, and ensure file integrity, it is recommended to use an archival tool (such as the open source 7zip) to compress the OVA/OVF. CenturyLink recommends 2-4GB increments.</p>
-* Calculate a checksum to help ensure the integrity of your files. Both Microsoft and several 3rd parties provide checksum calculation utilities; likewise a package exists for all major linux distros. In addition, a checksum can be generated by most archival programs (WinZip, WinRar, 7zip, etc.)</p>
-* Upload the OVF file to the appropriate datacenter. Make sure that the FTP server that you are uploading the OVF file to corresponds with the desired deployment datacenter, as this saves a great deal of time as the image does not need to be transferred for an additional time.  The specific FTP URL and account information used for uploading the templates will be provided to you at the time the service task is performed as part of a service engagement. <em>(Do not use the FTP User login information from Control Portal. This FTP service is not for template upload. For more information about this, please reference the [KB article about proper FTP usage](../Control Portal/ftp-users-in-control-portal.md)).</em>
-* Member servers may rely on domain controllers not present in the cloud 3 infrastructure to properly authenticate or execute policies- please take this into consideration when planning the upload priority, order and testing methodology. While not always possible, it is considered best practice disjoin the machine from the domain prior to exporting and then rejoin the domain once the import is complete, as the machine’s virtual hardware, SID and other descriptors will have changed.</p>
-* The same snapshot-based backup and restoration tools are available for imported templates but note these are not transaction-safe so application-aware backups should be utilized.  When deploying templates on a platform without snapshot capability (such as Hyperscale) then customer is responsible to developing and testing a backup and recovery process.
-* Describe any dependencies, specific applications to be tested, or other non-standard configurations to be tested during import.</p>
-* Please contact your account representative or email [noc@ctl.io](mailto:noc@ctl.io) to initiate a project importing the VM
+  * Validate the [Guest Operating System is supported on the CenturyLink Cloud for Import](../Service Taks/supported-guest-operating-systems-for-ovaovf-import.md)
```

Comment:
You are getting the error because you misspelled Tasks Also if you are in the same folder as the document you want to link to you can just do `./the-docuemnt-name.md`